src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -285,14 +279,14 @@ public bool IsTypeAttributeContext(CancellationToken cancellationToken)
             // cases:
             //    [ |
             //    class C { [ |
-            var token = this.TargetToken;
```

Comment:
please don't remove `this.`

Reply:
Because i wrote this code intentionally using that.  It helps me see what are instance lookups versus potential static calls off of a typename.

Reply:
You should have done `#pragma warning disable` to explicitly say, that this is intentional. Ok, i'll revert

Reply:
There are no warnings for these.

Reply:
I ment the same thing, but for suggestion. Or it isn't possible?